Sources/Parsers/AssetsCatalogParser.swift

```diff
@@ -104,6 +107,9 @@ extension AssetsCatalogParser {
     let type = item.extension ?? ""
 
     switch (type, AssetCatalog.Item.supported.contains(type)) {
```

Comment:
Remark about the code below on lin 110:

```
case ("", _), (_, true):
```

I think that was meant to handle when the folder either don't have an extension (`("", _)`, or have an extension just because it happens that the group/folder name contains a `.` in its name, but that has nothing to do with the extensions like `.colorset`, `.iconset` or other extensions we can encounter in the asset catalog structure. But that latter case doesn't match the behaviour tested by the second condition `(_, true)`, right?

- The case `(_, true)` is matched if `AssetCatalog.Item.supported.contains(type)` and given than `AssetCatalog.Item.supported` is an array containing the two extensions previously already tested (`colorset` and `imageset`) then those case are guaranteed to already be handled
 - And what we intended to do instead is for cases like a group named `"my.icons"` where `icons` would be considered as the extension while that folder should be considered as a standard group. But is that even possible (do asset catalog allow group names with `.` in them?), and if it's accepted, we should instead test the folder extension against _unsupported_ extensions (and I'm pretty sure that's what was done in the first version introducing that code before we decided to change from a blacklist to a whitelist, overlooking that change in the condition meaning doing that)

It would be nice to take advantage of that PR or do a separate one fix that bug while we're at it (test if `.` are allowed by Xcode and if they are either test against unsupported or think of a better logic)